mct_quantizers/pytorch/quantizer_utils.py

```diff
@@ -14,8 +14,12 @@
 # ==============================================================================
 from typing import Tuple
 
-import torch
 import numpy as np
+import torch
+
+from mct_quantizers.pytorch.constants import CPU, CUDA
+
+_device = CUDA if torch.cuda.is_available() else CPU
```

Comment:
What is the incentive for defining this global variable?
In my opinion, it is better to avoid such variables, and there is no need to enable device initialization from an outer scope.

Reply:
It's just a string the same way you used CUDA constant.
However, here we can set the device to cuda:3 if we want and the whole package is going to use that device. Very useful with shared servers.

Reply:
@haihabi 
Please take a look at this